### PR TITLE
Fuzzer: Adjust feature fuzzing frequency

### PR DESCRIPTION
**Pull request by kripken**
We used to fuzz MVP 1/3, all 1/3, and a mixture 1/3, but that gives far too much
priority to the MVP which is increasingly less important. It is also a good idea to
give "all" more priority as that enables more initial content to run (the fuzzer will
discard initial content if it doesn't validate with the features chosen in the current
iteration).

### CODE REVIEW

File: scripts/fuzz_opt.py [hunk 1]
```diff
@@ -126,16 +126,33 @@ def no_pass_debug():
...
     FEATURE_OPTS = CONSTANT_FEATURE_OPTS[:]
-    # 1/3 the time apply all the possible opts, 1/3 none of them, to maximize
-    # coverage both ways, and 1/3 pick each one randomly
-    if random.random() < 0.33333:
+
+    # Note that POSSIBLE_FEATURE_OPTS contains --disable-X for each feature, so
+    # we enable all features when we add nothing from it, etc. (assert on that
+    # form here to guard against future refactorings making a mess here).
+    assert '--disable' in POSSIBLE_FEATURE_OPTS[0]
+    assert '--enable' not in POSSIBLE_FEATURE_OPTS[0]
```
Comment (aheejin) [inline scripts/fuzz_opt.py:134]: How about renaming it to `POSSIBLE_DISABLE_FEATURE_OPTS` or something? Code like
```py
FEATURES_OPTS += POSSIBLE_FEATURE_OPTS
```
looks like we are enabling all the possible options out there when we are actually doing the very opposite.

**Review by aheejin — APPROVED** [review]

**Review by tlively — APPROVED** [review]
I like @aheejin's renaming idea, but otherwise LGTM

File: scripts/fuzz_opt.py (hunk 1 above)
Comment (kripken) [reply scripts/fuzz_opt.py]: Good idea, thanks, renamed and refactored.

**Comment by kripken** [comment]
Actually I may need to change the probabilities here, since I forgot that enabling all features disables parts of the fuzzer... for example enabling simd disables running d8 (since v128 on the boundary traps - we can't compare those values). I'll think about some solution to that first maybe.

**Pull request merged**